spec/features/scheduling/scheduled_publishing_spec.rb

```diff
@@ -0,0 +1,103 @@
+# frozen_string_literal: true
+
+RSpec.feature "Scheduled publishing" do
+  include ActiveSupport::Testing::TimeHelpers
+
+  around do |example|
+    Sidekiq::Testing.fake! do
+      travel_to(Time.zone.parse("2019-06-21"))
+      example.run
+      travel_back
+    end
+  end
+
+  scenario do
+    given_there_is_a_schedulable_edition
+    when_i_go_to_schedule_a_publishing
+    and_i_submit_the_reviewed_option
```

Comment:
So is this one :-).

Reply:
I went for `and_i_submit_a_review_option` in failed to indicate that the choice isn't significant